pootle/apps/pootle_store/models.py

```diff
@@ -1228,6 +1238,11 @@ def save(self, *args, **kwargs):
 
         self.pootle_path = self.parent.pootle_path + self.name
 
+        # Force validation of "name" field.
+        exclude_fields = list(set([f.name for f in self._meta.get_fields()]) -
+                              set(["name"]))
+        self.clean_fields(exclude=exclude_fields)
```

Comment:
I didn't had a deep look on all the failures there, but the first ones, the ones I checked, where because of some validation error on `last_sync_revision`.